indexing-service/src/test/java/org/apache/druid/indexing/common/task/batch/parallel/AbstractParallelIndexSupervisorTaskTest.java

```diff
@@ -183,8 +183,8 @@
           null
       );
 
-  protected static final double DEFAULT_TRANSIENT_TASK_FAILURE_RATE = 0.3;
-  protected static final double DEFAULT_TRANSIENT_API_FAILURE_RATE = 0.3;
+  protected static final double DEFAULT_TRANSIENT_TASK_FAILURE_RATE = 0.2;
```

Comment:
Is this a necessary change? seems unrelated to this problem

Reply:
Some tests failed because of timeout. As those tests pass with no issue in my computer, one reason that caused the timeout could be too often retries. Reducing failure rate can mitigate too often retry problem.